editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -4104,6 +4104,10 @@ void CanvasItemEditor::set_current_tool(Tool p_tool) {
 	_button_tool_select(p_tool);
 }
 
+void CanvasItemEditor::update_tool() {
```

Comment:
What's this method for and why does it have such generic name?
You can just call `set_current_tool()` directly.